stl/inc/variant

```diff
@@ -962,18 +962,14 @@ using _Variant_destroy_layer = conditional_t<conjunction_v<is_trivially_destruct
 #pragma clang diagnostic ignored "-Wdeprecated-volatile"
 #else // ^^^ Clang / not Clang vvv
 #pragma warning(push)
+#pragma warning(disable : 4244) // '%s': conversion from '%s' to '%s', possible loss of data

```

Comment:
Can [warning C4365](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-4-c4365?view=vs-2019) (signed/unsigned mismatch, off-by-default) be triggered here too?

Reply:
[Yes](https://godbolt.org/z/ao4pvt), good catch.